packages/action-menu/test/index.ts

```diff
@@ -436,56 +436,6 @@ export const testActionMenu = (mode: 'sync' | 'async'): void => {
 
             expect(firstRect).to.deep.equal(secondRect);
         });
-        it('opens and selects in a single pointer button interaction', async () => {
```

Comment:
Menu hover was removed for better a11y experience, so this test is no longer needed